schema/pet-insurance.json

```diff
@@ -1059,6 +1041,32 @@
                 "monthly-directdebit-card"
             ]
         },
+        "VetPaymentTerms": {
+            "type": "array",
+            "description": "List of payment terms.",
+            "minItems": 1,
+            "items": {
+                "$ref": "#/domains/PetInsurance/types/VetPaymentTerm"
+            }
+        },
+        "VetPaymentTerm": {
```

Comment:
Judging by the description, this is not a payment term, but fee limit. Consider renaming to `vetFeeLimit` or something more relevant.